src/pages/_app.tsx

```diff
@@ -160,7 +158,6 @@ export default function App({ Component, pageProps }: AppPropsWithLayout) {
       <Toaster />
 
       <CookiePrompt />
-      <MigrationAlert />
```

Comment:
`MigrationAlert` itself should be removed I assume?

Reply:
yep!